api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md

```diff
@@ -4,7 +4,7 @@ type: eventType
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Raised after a custom command item has been clicked and allows you to perform custom actions.
```

Comment:
```suggestion
Raised after a custom command item was clicked and allows you to perform custom actions.
```

Reply:
"to perform custom actions" this is a bit vague

Reply:
perhaps: "to implement the custom command's logic."

Reply:
Ok